atlasdb-docker-test-utils/src/main/java/com/palantir/atlasdb/testing/ProjectInfoMappings.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.testing;
+
+import java.net.InetAddress;
+import java.util.Map;
+
+import org.immutables.value.Value;
+
+import com.google.common.collect.Multimap;
+
+@Value.Immutable
+public interface ProjectInfoMappings {
+    Map<String, InetAddress> getHostToIp();
+    Multimap<String, String> getIpToHosts();
```

Comment:
String since InetAddress doesn't work well for a multimap key. (hashCode == -1, equals == false)